Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -112,6 +94,26 @@ extension UIViewController {
         CustomerIO.shared.performScreenTracking(onViewController: viewController)
     }
 
+    func getNameForAutomaticScreenViewTracking() -> String? {
+        let nameOfViewControllerClass = String(describing: type(of: self))
+
+        var name = nameOfViewControllerClass.replacingOccurrences(
+            of: "ViewController",
+            with: "",
+            options: .caseInsensitive
+        )
+        if name.isEmpty || name == "" {
+            if let title = title {
```

Comment:
Wouldn't it be a good idea to use guard let ? That'll make the code look cleaner.